### PR TITLE
troubleshooting symbols 

### PR DESCRIPTION
**Pull request by iogbole**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,6 +251,13 @@ in a few important ways:
 
 We're aiming that these limitations be improved on and eventually lifted in later versions.
 
+> **Note**
+>
+> If symbols are not displayed in Kibana after ingesting them, try restarting Kibana in your deployment (Cloud Cloud -> Deployments -> Kibana --> Force Restart).
```
Comment (rockdaboot) [inline README.md:256]:
```suggestion
> If symbols are not displayed in Kibana after ingesting them, try restarting Kibana in your deployment (Cloud -> Deployments -> <Deployment> -> Kibana --> Force Restart). 
```

File: README.md [hunk 2]
```diff
@@ -251,6 +251,13 @@ in a few important ways:
 
 We're aiming that these limitations be improved on and eventually lifted in later versions.
 
+> **Note**
+>
+> If symbols are not displayed in Kibana after ingesting them, try restarting Kibana in your deployment (Cloud Cloud -> Deployments -> Kibana --> Force Restart). 
+> This is a known issue in 8.5 and will be fixed in later versions.
```
Comment (rockdaboot) [inline README.md:257]: I guess you didn't mention 8.5.0 on purpose (to be not too specific) !?

**Comment by rockdaboot** [comment]
Maybe you want to add a screenshot like
Screenshot_20221101_111209

File: README.md (hunk 2 above)
Comment (iogbole) [reply README.md:257]: that's correct.

**Review by rockdaboot — APPROVED** [review]
LGTM

**Pull request merged**